examples/MMF-activity-recognition/mmf_utils.py

```diff
@@ -0,0 +1,105 @@
+from abc import ABC
+import json
+import logging
+import os
+import pickle
+import sys
+import tempfile
+import ast
+import torch
+import io
+import PIL
+from PIL import Image
+import tempfile
+import torchaudio
+import torchvision
+from omegaconf import OmegaConf
+
+from torchvision import transforms
+from torchvision.datasets.vision import VisionDataset
+
+from torchvision.io import (
+    read_video_timestamps,
+    read_video
+)
+logger = logging.getLogger(__name__)
+
+import csv
+from transformers import BertTokenizer
+import transforms as T
+PRE_TRAINED_MODEL_NAME = 'bert-base-uncased'
+
+BERT_MAX_LEN = 112
+
+bert_tokenizer_tfm = BertTokenizer.from_pretrained(PRE_TRAINED_MODEL_NAME).encode_plus
```

Comment:
Instead of doing all this, now you can use `build_processors` function from MMF to get all processors automatically built. You don't need to duplicate this code here now.